### DISCUSSION TITLE
2022-15: 新轮子 backon

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#347] [Announcements]
# 2022-15: 新轮子 backon

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-15/

**Comment by chenzx** [comment]
如果使用#Rust只写异步IO风格的代码，岂不是意味着根本不需要了解多线程编程下的各种引用类型的用法了？多线程编程对于计算密集型的确实需要（虽然可能不如GPU编程更实用），但对于IO密集的，异步编程就够用了吧
